apps/web/playwright/booking-limits.e2e.ts

```diff
@@ -266,9 +269,8 @@ test.describe("Booking limits", () => {
 
         // try to book directly via form page
         await page.goto(slotUrl);
-        await bookTimeSlot(page, { expectedStatusCode: 401 });
 
-        await expect(page.getByTestId("booking-fail")).toBeVisible({ timeout: 5000 });
+        await expectSlotNotAllowedToBook(page);
```

Comment:
Now, booking isn't possible(instead of a failure earlier) so we use different approach to verify it.